sonic-net/sonic-buildimage
# [Nokia][sonic-platform] Update Nokia sonic-platform submodule - induce NIF port shutdown at ungraceful reboot time

These changes provide for the automatic shutdown of NIF ports on LC when an ungraceful reboot scenario occurs.  Reboot and panic notifier hooks are now registered so that callback occurs from the kernel and NIF ports are subsequently shut down.

#### Why I did it

To facilitate the timely movement of traffic away from a crashed LC when its peers recognize that the associated links have gone down.

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it

Linux kernel reboot and panic notifier hooks are used to register a callback routine that, when invoked, stuffs all present transceiver modules into reset.

#### How to verify it

Cause an ungraceful reboot (whether via /usr/sbin/reboot or by causing a kernel panic) and verify that all LC native NIF links are brought down at reboot/panic time (on the way down).  It may be necessary to monitor the LC link peer(s) in order to verify in real-time.

#### Which release branch to backport (provide reason below if selected)

  TBD


#### Tested branch (Please provide the tested image version)





#### Description for the changelog

[Nokia][sonic-platform] Update Nokia sonic-platform submodule:  add reboot and panic notifier hooks to induce NIF port shutdown at ungraceful LC reboot time
